examples/diagnostics_and_criticism/Bayes_factor.ipynb

```diff
@@ -22,7 +22,7 @@
      "name": "stdout",
```

Comment:
<p>we could compare the value&nbsp;θ=0.5</p><p>&nbsp;(representing no bias) against <s>an</s> the result from a model were we let</p><p> </p><p>also add a reference to the paper.</p><p>I'd also say that this is a special case where the math works out conveniently to just having to compute the ratio between prior and posterior.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>